### ISSUE TITLE
the results on MNIST

### ISSUE THREAD

**Issue by catherinezll95** [samet-akcay/ganomaly#25]
Hello, could you tell me whether the results of MNIST in your paper were obtained without verification set? Thank you.

**Comment by samet-akcay** [comment]
Hi, the dataset is only split into train and test set. There is no validation set within the experimentation. The test set is only used to validate the performance of the model after every epoch. 

I’m closing this due to inactivity. Please reopen if you still encounter this issue with the latest version.
